### DISCUSSION TITLE
"code actions" only works with master?

### DISCUSSION THREAD

**Discussion by botkero** [squidfunk/mkdocs-material#5012] [Q&A] [answered]
I wanted to add the code actions for my page, but I noticed that the default branch has to be master. I wanted to ask here if it is possible to adjust this? 

https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#code-actions

**Comment by squidfunk** [comment, marked as answer]
No, they should work on other branches. You may have misconfigured your project. If you provide a [reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), we can look into it.

**Reply by botkero** [reply]
Project is public. I hope that is enough?
https://github.com/botkero/edf5

**Reply by squidfunk** [reply]
Well, code actions (edit) are shown? I'm not sure what the problem is.

<img width="965" alt="Bildschirmfoto 2023-02-09 um 09 37 52" src="https://user-images.githubusercontent.com/932156/217759944-dd85fba3-6b42-4cd2-9bdc-b2034c09288d.png">

**Reply by botkero** [reply]
> Well, code actions (edit) are shown? I'm not sure what the problem is.
> 
> <img alt="Bildschirmfoto 2023-02-09 um 09 37 52" width="965" src="https://user-images.githubusercontent.com/932156/217759944-dd85fba3-6b42-4cd2-9bdc-b2034c09288d.png">

As I said, this leads to the wrong branch and not to main.

**Reply by squidfunk** [reply]
Please see [`edit_uri`](https://www.mkdocs.org/user-guide/configuration/#edit_uri):

``` yaml
edit_uri: blob/main/docs/
```
